docs/release/release-actions-0.6.0.md

```diff
@@ -0,0 +1,55 @@
+# Release Manager Actions Checklist 0.6.0
```

Comment:
I'm not sure it's necessary to create this page for the next release now, given that it won't be used for a while. I'd be inclined to let the release manager for the next release create this from the template as a first step in the release procedure.

Reply:
Sounds good. I will delete this.